aspnet/2-structured-data/Models/DbBookStore.cs

```diff
@@ -46,12 +46,13 @@ public void Delete(long id)
         // [START list]
         public BookList List(int pageSize, string nextPageToken)
         {
-            var pageOfBooks = (null == nextPageToken ?
-                (from book in _dbcontext.Books orderby book.Id select book) :
-                (from book in _dbcontext.Books
-                 where book.Id > Int32.Parse(nextPageToken)
-                 orderby book.Id
-                 select book)).Take(pageSize + 1);
+            IQueryable<Book> query = from book in _dbcontext.Books orderby book.Id select book;

```

Comment:
Every EF7 tutorial that I can find uses lambda-style query syntax

 - [Introducing view components](https://docs.asp.net/projects/mvc/en/latest/views/view-components.html)
 - [Unit Testing ASP.NET 5 controllers with Entity Framework 6 in-memory database](http://www.jerriepelser.com/blog/unit-testing-aspnet5-entityframework7-inmemory-database)
 - [Build an ASP.NET 5 Web API EF7 Code First Step-by-Step](http://tutaurelia.net/2016/01/02/build-an-asp-net-5-web-api-ef7-code-first-step-by-step/)
 - [ASP.NET 5 and AngularJS Part 4, Using Entity Framework 7](http://stephenwalther.com/archive/2015/01/17/asp-net-5-and-angularjs-part-4-using-entity-framework-7)
 - [A Look at ASP.NET 5: Part 3 - EF7](http://wildermuth.com/2015/3/17/A_Look_at_ASP_NET_5_Part_3_-_EF7)

^ misc Google results

Looking for usage of EF7 in https://github.com/aspnet/Docs also shows consistent use of the lambda-style syntax:

 - [BooksController of Your First ASP.NET Application sample](https://github.com/aspnet/Docs/blob/master/aspnet/tutorials/your-first-aspnet-application/sample/src/ContosoBooks/Controllers/BooksController.cs#L21)
 - [MoviesController of First MVC App sample](https://github.com/aspnet/Docs/blob/master/mvc/getting-started/first-mvc-app/start-mvc/sample/src/MvcMovie/Controllers/MoviesController.cs#L32)

IMO this syntax is more popular with up-and-coming documentation (EF7).  **BUT** query syntax is more idiomatic for EF6.

So.  I think you can make a good argument that query syntax is appropriate for EF6 samples.  That said, it results in more code.

Thoughts?  /cc @jsimonweb